Clarify AWS profiles

ttung opened the pull request:
### Description
src/dst is confusing.  Make it explicitly covidhub vs aspen.  This makes it easier to manage multiple connections, which we will need to import to a staging db hosted on AWS.

### Test plan
None.

phoenixAja commented on src/py/aspen/cli/db.py:
```diff
@@ -37,31 +37,28 @@ def interact(ctx, profile):
 
 
 @db.command("import-covidhub-project")
+@click.option("--covidhub-aws-profile", type=str, required=True)
```
wouldn't this always be the biohub aws profile?

phoenixAja replied:
or maybe i'm missing what you mean by `covidhub-aws-profile` exactly, or what the issue is that this PR is fixing?

ttung replied:
It is always the biohub aws profile.  We can't hardcode it though because everyone could name it differently.

phoenixAja replied:
makes sense

phoenixAja commented on src/py/aspen/covidhub_import/implementation.py:
```diff
@@ -48,8 +48,10 @@
-    interface = Cinit_db(Cutil.get_db_uri(secret_id))
+def covidhub_interface_from_secret(
+    covidhub_aws_profile: str, secret_id: str
+) -> CSqlAlchemyInterface:
+    interface = Cinit_db(Cutil.get_db_uri(secret_id, aws_profile=covidhub_aws_profile))
```
does the leading C stand for covidhub? lol

phoenixAja replied:
i feel like `CUtil` should be `c_util` and `Cinit_db` should be `c_init_db`, that seems more python-ish to me. That's a nitpick though

ttung replied:
I'll just be less lazy and call it covidhub_*.

phoenixAja replied:
haha even better!

Commit:
Clarify AWS profiles

### Description
src/dst is confusing.  Make it explicitly covidhub profile, whereas aspen profile is the default one  This makes it easier to manage multiple connections, which we will need to import to a staging db hosted on AWS.

### Test plan
None.
